### PR TITLE
Workaround for ternary operator bug in vs2015

### PR DESCRIPTION
**Pull request by umar456**
The ternary operator was not working correctly in vs2015.

build arrayfire windows ci

### CODE REVIEW

File: src/backend/cuda/kernel/harris.hpp [hunk 1]
```diff
@@ -312,9 +312,8 @@ void harris(unsigned* corners_out,
     memFree(d_responses);
     memFree(d_corners_found);
 
-    *corners_out = (max_corners > 0) ?
-                   min(corners_found, max_corners) :
-                   min(corners_found, corner_lim);
+    if (max_corners > 0) { *corners_out = min(corners_found, max_corners); }
+    else                 { *corners_out = min(corners_found, corner_lim);  }
```
Comment (pavanky) [inline src/backend/cuda/kernel/harris.hpp]: `*corners_out = min(corners_found, max_corners > 0 ? max_corners : corner_lim)`

Comment (umar456) [inline src/backend/cuda/kernel/harris.hpp]: I just want to avoid ternaries altogether.

Comment (pavanky) [inline src/backend/cuda/kernel/harris.hpp]: alright

**Comment by 9prady9** [comment]
Aren't there other places that uses ternary operator in similar fashion ?

**Comment by umar456** [comment]
build arrayfire windows ci

**Comment by umar456** [comment]
build arrayfire windows ci

**Pull request merged**